aeron-driver/src/main/c/aeron_publication_image.c

```diff
@@ -384,8 +384,9 @@ void aeron_publication_image_track_rebuild(aeron_publication_image_t *image, int
             }
         }
 
-        const int64_t rebuild_position = *image->rcv_pos_position.value_addr > max_sub_pos ?
-            *image->rcv_pos_position.value_addr : max_sub_pos;
+        const int64_t rcv_position = *image->rcv_pos_position.value_addr;
+        const int64_t consumed_position = rcv_position < min_sub_pos ? rcv_position : min_sub_pos;
```

Comment:
Maybe this should be
```suggestion
        const int64_t consumed_position = INT64_MAX == min_sub_pos ? rcv_position : min_sub_pos;
```
So that if we have a valid min sub pos and its beyond the rcv position we use it instead